src/pgwire/src/protocol.rs

```diff
@@ -1505,6 +1542,9 @@ where
         get_response: GetResponse,
         fetch_portal_name: Option<String>,
         timeout: ExecuteTimeout,
+        statement_kind: Option<StatementKind>,
+        isolation_level: IsolationLevel,
```

Comment:
isolation level is part of the session which `self` has access to, so shouldn't need to be passed.